File: 2022-1/ALGORITMOS/TDE_26:05/ex3.c
#include <stdio.h>

int main(){
    float num [10],quadrado[10];
    int i;
    for ( i = 0; i < 10; i++){
        printf("Digite os números:\n");
        scanf("%f%*c",&num[i]);
    }
    for(i=0;i<10;i++){
        printf("valor digitado:%.2f\n",num[i]);
    }
    for ( i = 0; i < 10; i++){
        quadrado[i]=num[i]*num[i];
        printf("quadrado:%.2f\n",quadrado[i]);
    }

    
}

File: 2023-2/Estrutura de Dados/aula3/TDE/ex3.py
def encontrar_maximo_minimo(lista):
    maximo = lista[0]
    minimo = lista[0]

    for num in lista:
        if num > maximo:
            maximo = num
        if num < minimo:
            minimo = num

    return maximo, minimo

def obter_lista():
    entrada = input("Digite números inteiros separados por espaço: ")
    numeros = entrada.split(" ")

    for num in numeros:
        try:
            numeros.append(int(item))
        except ValueError:
            print(f"Valor inválido: {item}. Ignorando este valor.")
    return numeros